feat: add initial commit for Python evaluation client using Rust core (#2358)
flipt-io/flipt: sdk/client/python/README.md

### Instructions

To use this client, you can run the following command from the root of the repository:

```bash
cargo build
```

> This should generate a `target/` directory in the root of this repository, which contains the dynamic linking library built for your platform. This dynamic library will contain the functinonality necessary for the Python client to make FFI calls. Depending on what your platform is you can set the `ENGINE_LIB_PATH` to:

Reviewer: ```suggestion
This should generate a `target/` directory in the root of this repository, which contains the dynamic linking library built for your platform. This dynamic library will contain the functionality necessary for the Python client to make FFI calls. You'll need to set the `ENGINE_LIB_PATH` environment variable depending on your platform:
```
